gcp-observability/src/test/java/io/grpc/gcp/observability/ObservabilityConfigImplTest.java

```diff
@@ -42,6 +42,7 @@
   private static final String LOG_FILTERS = "{\n"
       + "    \"enable_cloud_logging\": true,\n"
       + "    \"destination_project_id\": \"grpc-testing\",\n"
+      + "    \"flush_message_count\": \"1000\",\n"
```

Comment:
Hmmm if `flush_message_count` is meant to be a `long` (basically an integral type) then why is it in double quotes? Will this work

```
flush_message_count: 1000
```

instead of what you have

```
flush_message_count: "1000"
```

Reply:
I noticed this after my initial commit and made the changes.

Also, since we use [`JsonUtil.getNumberAsLong()`](https://github.com/grpc/grpc-java/blob/master/core/src/main/java/io/grpc/internal/JsonUtil.java#L154), it works even when `flush_message_count` value is `String`.

Reply:
> even when flush_message_count value is String.

That's good to know so it is more flexible from user point of view